smart-cards/src/main/java/gov/va/api/health/smartcards/WellKnownController.java

```diff
@@ -21,6 +21,11 @@ String jwks() {
     return jwksProperties.jwksPublicJson();
   }
 
+  @GetMapping(value = "/r4/.well-known/jwks.json")
```

Comment:
This can be a second value on the method above.

Also, since we have dedicated dstu2 endpoints for metadata and open-api, should we add one here as well?

Reply:
On further reflection, how about `{"/.well-known", "/dstu2/.well-known", "/r4/.well-known"}` on the class level and then `"/jwks.json"` and ` "/smart-configuration"` for the two methods